lix/packages/sdk/src/discussion/start-discussion.ts

```diff
@@ -0,0 +1,68 @@
+import { Kysely } from "kysely";
+import type { LixDatabaseSchema } from "../database/schema.js";
+import { v4 } from "uuid";
+
+export async function startDiscussion(args: {
```

Comment:
Minor: `create` sounds more natural than `start`. besides, here too to avoid ambiguity problems down the road, i would call the function `createChangeDiscussion`.